services/apache.cf

```diff
@@ -2,31 +2,76 @@
 bundle common apache
 {
     vars:
-        "configdir" string => "/opt/apache";
 
-        "performance_profile" string => "l_4c8m";
+        any::
+            "configdir" string => "/opt/apache";
 
-        "service_name" string => "apache2";
-        "daemon_name" string => "apache2";
+            "performance_profile" string => "l_4c8m";
 
-        "restart" string => "$(paths.path[systemctl]) restart $(service_name)";
+            "service_name" string => "apache2";
+            "restart" string => "$(paths.path[systemctl]) restart $(service_name)";
 
-        "packages"  data => parsejson('
-            {
-                "install": {
-                    "apache2": "",
+        sles::
+            "daemon_name" string => "httpd-worker";
+
+            "packages"  data => parsejson('
+                {
+                    "install": {
+                        "apache2": "",
+                        "apache2-worker": "",
+                        "apache2-utils": ""
+                    },
+                    "remove": {
+                        "apache2-prefork": ""
+                    }
+                }
+            ');
+
+            "template_2_destination" data => parsejson('
+                {
+                    "suse_apache_sysconfig.mustache": "/etc/sysconfig/apache2",
+                    "start_apache2.mustache": "/usr/sbin/start_apache2",
+                    "httpd.conf.mustache": "$(configdir)/httpd.conf",
+                    "ssl.conf.mustache": "$(configdir)/ssl.conf",
+                    "server.conf.mustache": "$(configdir)/server.conf"
+                }
+            ');
+
+        debian::
+            "daemon_name" string => "apache2";
+
+            "packages"  data => parsejson('
+                {
+                    "install": {
+                        "apache2": "",
+                        "php7.3-fpm": "",
+                        "php7.3-mbstring": "",
+                        "php7.3-zip": "",
+                        "php7.3-gd": "",
+                        "php7.3-curl": "",
+                        "php7.3-mysql": "",
+                        "php7.3-pgsql": "",
+                        "php7.3-ldap": "",
+                        "php7.3-xml": "",
+                        "php-xdebug": "",
+                        "php7.3-bcmath": "",
+                        "php7.3-imap": "",
+                        "php7.3-json": "",
+                        "php7.3-soap": ""
+                    }
```

Comment:
Leave out the `php` packages as before. This can be installed in a seperate `json` file if neeeded.

Reply:
Woops, my mistake, removed the packages and pushed to mu branch.